### DISCUSSION TITLE
How to set hide series default for angular?

### DISCUSSION THREAD

**Discussion by manhtq1197** [apexcharts/apexcharts.js#2627] [Q&A]
I have seen custom toggleSeries method of the JS but not found the Anglular custom. Can someone help me set the default value when displaying the chart? plz

![image](https://user-images.githubusercontent.com/71320797/133395352-e311f232-5ddd-4c3d-8936-2692cb41d9f8.png)

**Comment by acarmona1989** [comment]
Hi.
There is a method for hiding series. https://apexcharts.com/docs/methods/#hideSeries

The chart must be rendered before execute this method, otherwise the component will throw an exception. Also if you have access to the chart instance you could invoke hideSeries method directly.

So far, there is not a property related to the feature hide series by default

**Reply by andreajsfullstack** [reply]
Not work
